src/test/run-pass/crate-path-absolute.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(crate_in_paths)]
+
+use crate::m::f;
+
+mod m {
+    pub fn f() {}
+    pub fn g() {}
+
+    // OK, visibilities are implicitly absolute like imports
+    pub(in crate::m) struct S;
+}
+
+fn main() {
+    f();
```

Comment:
Not that I think it's *really* needed, bu this test doesn't demonstrate that things resolved correctly in any way. Perhaps have `f` and `g` return values, and we can assert that you get the right thing back?